tests/autoyast/installation.pm

```diff
@@ -305,6 +304,12 @@ sub run {
 
     # Cannot verify second stage properly on s390x, so reconnect to already installed system
     if (check_var('ARCH', 's390x')) {
+        # Due to bsc#1167210, the system is shut down on reboot on zkvm on SP2.
+        # This is workaround to connect to the shut off machine.
+        if (is_sle('>=15-sp2')) {
+            console('installation')->disable_vnc_stalls;
+            assert_shutdown_and_restore_system('reboot', 120);
```

Comment:
Let's record soft-failure here, not to forget to update it once issue is resolved.